apps/inntektsmelding-stub/src/main/java/no/nav/registre/inntektsmeldingstub/util/XmlInntektsmelding201809.java

```diff
@@ -57,6 +57,10 @@
 
 public class XmlInntektsmelding201809 {
 
+    private XmlInntektsmelding201809() {
+        throw new IllegalStateException("Utility klasse");
```

Comment:
Synes det er litt overkill med en exception. Det vill uansett være umulig å kalle denne konstruktøren uten compiler feil.

Reply:
Dette er det samme for alle andre av denne type fix:)